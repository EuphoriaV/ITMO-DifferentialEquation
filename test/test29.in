1.51
(((x * ((6.38 + 1.99) * (8.37 + ((x * 10.5) * (x + x))))) * (4.98 + (10.3 * x))) + ((5.05 + (4.35 - x)) * (4.24 + x)))
3.14
4.85