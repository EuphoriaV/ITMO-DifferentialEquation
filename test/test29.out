3071.7595